src/libcsg/modules/io/dlpolytopologyreader.cc

```diff
@@ -224,6 +227,17 @@ bool DLPOLYTopologyReader::ReadTopology(string file, Topology &top) {
         if (!top.BeadTypeExist(beadtype)) {
           top.RegisterBeadType(beadtype);
         }
+
+        Elements elements;
```

Comment:
elements object should be created outside for loop

Reply:
Done.